src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
```

Comment:
Link to a stylesheet should have an extension `.css`, not `.scss`. SCSS is a preprocessor scripting language that will compile down to CSS, and browsers don't understand SCSS syntax.